CanHazFunny/CanHazFunny/Jester.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data.SqlTypes;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace CanHazFunny
+{
+    public class Jester
+    {
+        private readonly JokeOutput _JokeOutput;
+        private readonly JokeService _JokeService; 
+        public Jester(JokeOutput jokeOutput, JokeService jokeService) {
+            _JokeOutput = jokeOutput ?? throw new ArgumentNullException(nameof(jokeOutput));
+            _JokeService = jokeService ?? throw new ArgumentNullException(nameof(jokeService));
+        
+        }
+
+        public static bool ContainsChuckNorris(string joke) {
+            return joke?.IndexOf("Chuck Norris", StringComparison.OrdinalIgnoreCase) > -1; 
```

Comment:
Interesting approach to this, but using .Contains() might be a simpler and more readable approach
```suggestion
            return joke?.Contains("Chuck Norris"); 
```